pom.xml

```diff
@@ -53,6 +55,32 @@
       </dependency>
     </dependencies>
   </dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.13.4.1</version>
+    </dependency>
```

Comment:
Remove this in favor of the Jenkins API plugin org.jenkins-ci.plugins:jackson2-api